conf/messages.en

```diff
@@ -392,7 +392,7 @@ partnership.registrationAvailableSoon.header = Partnership
 partnership.registrationAvailableSoon.title = Partnership registration will be available soon
 partnership.registrationAvailableSoon = {0} and we’ll contact you when the service becomes available.
 partnership.registrationAvailableSoon.link = Tell us you would like to register as a partnership
-partnership.registrationAvailableSoon.href = https://www.gov.uk/guidance/check-if-you-need-to-register-for-plastic-packaging-tax
+partnership.registrationAvailableSoon.href = mailto:pptpartnerships@hmrc.gov.uk
```

Comment:
Although presumably this is what "content" have requested, I think I would like to see the email address reflected in the page config.

Reply:
Also, is it worth extracting that email address into the app config so that it might be changed without having to cut a new release of the service?

Reply:
Probably not as I think Adam suggested this will likely be in place for all of 2 weeks.